### PR TITLE
Register lambda

### PR DESCRIPTION
**Pull request by sbendarsky**

### CODE REVIEW

File: backend/lambda_functions/registartion_form/registration_form_lambda.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import json
+import psycopg2
+import os
+import boto3
+
+# Environment Variables
+REGION = os.environ['REGION']
```
Comment (entrypoint-community) [inline backend/lambda_functions/registartion_form/registration_form_lambda.py:7]: love that!

File: backend/lambda_functions/registartion_form/registration_form_lambda.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    except Exception as e:
+        print(e)
+        return {
+            'statusCode': 500,
+            'body': json.dumps('Internal Server Error')
+        }
+    
+    # Send the email to the user
+    try:
+        response = ses.send_email(
```
Comment (entrypoint-community) [inline backend/lambda_functions/registartion_form/registration_form_lambda.py:49]: please make this a function

File: backend/lambda_functions/registartion_form/registration_form_lambda.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+def lambda_handler(event, context):
+    # extract the data from the event
+    user_data = json.loads(event['body'])
+    name = user_data['name']
+    email = user_data['email']
+    phone = user_data['phone']
+
+    # Connect to the database
+    try:
+        conn = psycopg2.connect(
```
Comment (entrypoint-community) [inline backend/lambda_functions/registartion_form/registration_form_lambda.py:21]: please create a connection function

File: backend/lambda_functions/registartion_form/registration_form_lambda.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+            host=DB_HOST,
+            database=DB_NAME,
+            user=DB_USER,
+            password=DB_PASSWORD,
+            port=DB_PORT
+        )
+        cursor = conn.cursor()
+
+        # Inserting userdata into db (assumes table is already created and called community_members)
```
Comment (entrypoint-community) [inline backend/lambda_functions/registartion_form/registration_form_lambda.py]: think about creating a general execution method for later function development (maybe more queries?)

File: backend/lambda_functions/registartion_form/registration_form_lambda.py (hunk 1 above)
Comment (sbendarsky) [reply backend/lambda_functions/registartion_form/registration_form_lambda.py:7]: thanks bobi

**Review by entrypoint-community — APPROVED** [review]

**Pull request merged**